cli/src/commands/publish.ts

```diff
@@ -0,0 +1,233 @@
+import { Command } from "commander";
+import chalk from "chalk";
+import { select, confirm, password } from "../prompts.js";
+import {
+  createSpinner,
+  printSuccess,
+  printError,
+  printNodeInfo,
+  symbols,
+  maskString,
+} from "../ui.js";
+import { getApiKey, getEnvConfig, getPrivateKey, setPrivateKey } from "../config.js";
+import {
+  getDraftNode,
+  listNodes,
+  publishNode,
+  signerFromPkey,
+} from "@desci-labs/nodes-lib/node";
+
+export function createPublishCommand(): Command {
+  return new Command("publish")
+    .description("Publish a node to Codex (requires signing)")
+    .argument("[node]", "Node UUID or partial UUID")
+    .option("-k, --private-key", "Prompt for private key (won't be saved)")
```

Comment:
I think it would be better to prompt for it, so it doesn't end up in the HISTFILE :eyes: